README.md

```
@@ -580,9 +581,13 @@ CI automatically validates:
 | Bundle consistency | ✅ | ✅ |
 | Manifest sync | ✅ | ✅ |
 | Thumbnails | ✅ | ❌ (optional) |
+| ComfyUI node compatibility | ✅ (informational PR comment) | ❌ |
 
 Run locally before committing:
 ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)